packages/@react-spectrum/table/src/TableViewBase.tsx

```diff
@@ -690,6 +691,17 @@ function TableVirtualizer<T>(props: TableVirtualizerProps<T>) {
   );
 }
 
+function renderChildren<T extends object>(parent: View | null, views: View[], renderWrapper: TableVirtualizerProps<T>['renderWrapper']) {
```

Comment:
Is there a way to export the version of this (renderChildren()) in Virtualizer and use it here? The method signature seemed to be the only change. Or is there a reason we don't do that?

Reply:
We could but I was trying to minimize the amount of "deprecated" things we export. The way RAC does it is the preferred way, but we need the old way in RSP for now. Hopefully in the future we can update RSP accordingly.